xarray/core/pycompat.py

```diff
@@ -79,3 +81,8 @@ def is_dask_collection(x):
 
 def is_duck_dask_array(x):
     return is_duck_array(x) and is_dask_collection(x)
+
+
+def is_chunked_array(x):
```

Comment:
Could this look for a `chunks` attribute instead of a hardcoded list of modules that are know to support chunked arrays?

Reply:
That's a much better idea

Reply:
Changed in https://github.com/pydata/xarray/pull/7019/commits/b1a4e35c4153334032b0ec3337f870c4a5992caf